airflow/config_templates/default_airflow.cfg

```diff
@@ -330,6 +330,8 @@ cookie_secure = False
 # Set samesite policy on session cookie
 cookie_samesite =
 
+# Default setting for wrap toggle on DAG code and TI log views.
+default_wrap = True
```

Comment:
Can we change the default setting to be `False`, to keep the current default behavior

Reply:
That sounds fine to me but I'm not at my dev env so I don't have the ability to commit + squash that this moment